diff --git a/demo/realtime-img2img/routes/parameters.py b/demo/realtime-img2img/routes/parameters.py
index 77f8755..7a6431b 100644
--- a/demo/realtime-img2img/routes/parameters.py
+++ b/demo/realtime-img2img/routes/parameters.py
@@ -75,6 +75,8 @@ async def update_params(request: Request, app_instance=Depends(get_app_instance)
             params["num_inference_steps"] = int(data["num_inference_steps"])
         if "seed" in data:
             params["seed"] = int(data["seed"])
+        if "negative_prompt" in data:
+            params["negative_prompt"] = str(data["negative_prompt"])
         if "t_index_list" in data:
             t_index_list = data["t_index_list"]
             if isinstance(t_index_list, list) and all(isinstance(x, int) for x in t_index_list):
